packages/felles/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@navikt/fp-felles",
```

Comment:
Dette pakke-navnet er litt for generelt. Med eit sånt navn i ei pakke på toppnivå er det vanskeleg å sjå kva som passar inn her. Da blir det fort eit bøtteprosjekt for alt mogleg rart. 

Mogleg ein burde legga denne i ft-repoet, i ui-komponenter. Evt ha ein lokal versjon av dette prosjektet. Men veldig åpen for bedre navn enn det

Reply:
jeg er enig i at navnet er for generelt. Grunnen til at jeg la den her og ikke i ft-* er fordi den bruker `hentDokumentLenke` som er fp-spesifikk. Jeg tenker at så lenge den ikke direkte skal brukes av sif så er det litt rart om vi legger fp-komponenter der nå vi ber dem om å flytte flere av komponentene over til sitt eget repo.

Men jeg er veldig med på å lage en lokal versjon av `fp-ui-komponenter` 😄 Kan fikse det før jeg merger!

Reply:
Ja einig i det du seier om å flytta til ft-repoet